android/FirebasePlugin.java

```diff
@@ -304,6 +308,7 @@ public void onPause() {
   }
 
   public void onStop() {
+    stopTrace();
```

Comment:
does this work ?

Reply:
It will get called, every time user stopping the game using home button. this will be ideal for us.